identity/handler_test.go

```diff
@@ -676,19 +676,20 @@ func TestHandler(t *testing.T) {
 	})
 
 	t.Run("case=should fail to update identity if input json is empty or json file does not exist", func(t *testing.T) {
-                for name, ts := range map[string]*httptest.Server{"public": publicTS, "admin": adminTS} {
-                        t.Run("endpoint="+name, func(t *testing.T) {
-                                        var cr identity.AdminCreateIdentityBody
-                                        cr.SchemaID = "employee"
-                                        cr.Traits = []byte(`{"email":"` + x.NewUUID().String() + `@ory.sh", "department": "ory"}`)
-                                        res := send(t, ts, "POST", "/identities", http.StatusCreated, &cr)
-
-                                        id := res.Get("id").String()
-                                        res = send(t, ts, "PUT", "/identities/"+id, http.StatusBadRequest, nil)
-                                        assert.Contains(t, res.Get("error.reason").String(), `Empty input json or file does not exist`, "%s", res.Raw)
-                        })
-                }
-        })
+		for name, ts := range map[string]*httptest.Server{"public": publicTS, "admin": adminTS} {
+			t.Run("endpoint="+name, func(t *testing.T) {
+				var cr identity.AdminCreateIdentityBody
+				cr.SchemaID = "employee"
+				cr.Traits = []byte(`{"email":"` + x.NewUUID().String() + `@ory.sh", "department": "ory"}`)
+				res := send(t, ts, "POST", "/identities", http.StatusCreated, &cr)
+
+				id := res.Get("id").String()
+				res = send(t, ts, "PUT", "/identities/"+id, http.StatusBadRequest, nil)
+				assert.Contains(t, res.Get("error.reason").String(), `Unable to decode HTTP Request Body because its HTTP `+
```

Comment:
Hi @Benehiko , this version of the test case still uses assert.Contains() method only. 
#### Reason: 
As you can see assert.ErrorIs(t, **err,** session.ErrIdentityDisabled)  method looking for **error type** variable as **second parameter**.   Here res is of type gJson.Result, and I could not find any method for this struct that returns error output. Link that I referred : https://sourcegraph.com/github.com/tidwall/gjson/-/docs#Result. 

Please correct me if I am missing something. 

Thanks.

Reply:
Yes you are right, this is perfectly fine :+1:

Reply:
ah i see, then it's fine like this - thank you :)